src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/læremidler/dto/VedtakLæremidlerDto.kt

```diff
@@ -29,10 +30,18 @@ sealed interface VedtakLæremidlerRequest : VedtakRequest
 sealed interface VedtakLæremidlerResponse : VedtakResponse
 
 data class VedtaksperiodeLæremidlerDto(
+    val id: UUID,
     val fom: LocalDate,
     val tom: LocalDate,
+    val status: VedtaksperiodeStatus? = null,
 )
 
-fun List<Vedtaksperiode>.tilDto() = this.map { VedtaksperiodeLæremidlerDto(fom = it.fom, tom = it.tom) }
+enum class VedtaksperiodeStatus {
+    NY,
+    ENDRET,
+    UENDRET,
+}
```

Comment:
Disse har ikke med `FJERNET`, husket ikke om det var noe dere hadde snakket med Lars om? 
Er det behov for det / Burde det vises?

Reply:
Det stemmer! Det er en litt større oppgave å fikse, fordi vi da må passe på at perioder med status SLETTET ikke blir brukt i beregning og liknende. 

Jeg lager en oppgave! 🍀